engine/src/mblstore.cpp

```diff
@@ -74,8 +74,10 @@ static struct {const char *name; MCPurchaseState state;} s_purchase_states[] =
     {"unverified", kMCPurchaseStateUnverified},
 };
 
-// we maintain here a list of known pending purchases
+// we maintain here a list of known pending purchases, and a list of completed purchases
 static MCPurchase *s_purchases = nil;
+#error "NEed to finish"
```

Comment:
Still need to add a cleanup function for s_completed_purchases...
Not suitable for merging as for now